archunit/src/main/java/com/tngtech/archunit/core/domain/AccessTarget.java

```diff
@@ -317,6 +317,12 @@ private Functions() {
             this.returnType = builder.getReturnType();
         }
 
+        @Override
+        @SuppressWarnings({"unchecked", "rawtypes"}) // cast is okay, since this list can only be used in a covariant way (immutable)
+        public List<JavaType> getParameterTypes() {
+            return (List) parameters;
```

Comment:
As far as I could see, we can replace the implementation with:
```
            throw new RuntimeException("No test breaks if we do this!");`
```
😜

Reply:
True, it's not rocket science :wink: , but since this is a library and also for documentation it makes sense to still test this I guess.